Barotrauma/BarotraumaShared/Source/DebugConsole.cs

```diff
@@ -419,12 +419,6 @@ static DebugConsole()
                 });
             }));
 
-            commands.Add(new Command("togglekarma", "togglekarma: Toggles the karma system.", (string[] args) =>
-            {
-                if (GameMain.Server == null) return;
-                GameMain.Server.KarmaEnabled = !GameMain.Server.KarmaEnabled;
-            }));
-
```

Comment:
I'd recommend reverting this, since you'd probably want to be able to toggle karma on a dedicated server.

Reply:
Alright, gimme a sec